src/NationalInstruments.Analyzers/Style/ChainOfMethodsWithLambdasAnalyzer.cs

```diff
@@ -36,25 +36,52 @@ public override void Initialize(AnalysisContext context)
             context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
 
             context.RegisterSyntaxNodeAction(
-                AnalyzeSyntaxNode,
+                AnalyzeSyntaxContext,
                 SyntaxKind.ExpressionStatement,
                 SyntaxKind.EqualsValueClause,
                 SyntaxKind.Argument,
                 SyntaxKind.ArrowExpressionClause);
+
+            context.RegisterSyntaxNodeAction(
+                AnalyzeArrayInitializerContext,
+                SyntaxKind.ArrayInitializerExpression);
         }
 
-        private static void AnalyzeSyntaxNode(SyntaxNodeAnalysisContext context)
+        private static void AnalyzeSyntaxContext(SyntaxNodeAnalysisContext context)
```

Comment:
`AnalyzeSyntaxContext` is generic and really an "action" that accepts a "context" as its parameter. I'd rename it to something like `AnalyzeExpression` or something more fitting for its `SyntaxKind`s

Reply:
Fixed [here](https://github.com/ni/csharp-styleguide/pull/65/commits/d2079a7f6644f6a39faa54aa6fa294d8552725c9).